libgpos/include/gpos/task/CAutoTaskProxy.h

```diff
@@ -58,7 +58,7 @@ namespace gpos
 			CEvent m_event;
 
 			// propagate error of sub-task or not
-			BOOL m_fPropagateError;
+			BOOL m_fPropagateSubTaskError;
```

Comment:
My understanding from the comment is that
1. should I propagate error / exception from subtask
or
2. Eat the error / exception and continue.

For e.g. Let say we have task t1 and task t2 running on different thread. So if t1 wait for t2 and t2 throw exception, then with this flag, we can decide whether to propagate that from t2 to t1. Thats how this flag has been used and tested in unit test.

At line #569 in CAutoTaskProxy, PropageSubTaskError is called. This is calling function 'CheckSubTaskError' (old name : CheckError). CheckSubTaskError / CheckError is always called from wait function.

I have two commits. 1. renaming and another 2. propagating error. Do you want me to open two different pull request ?